analyses/bangladesh/trigger_development/read_in_data.py

```diff
@@ -13,11 +13,16 @@
 from src.indicators.flooding.glofas import glofas
 from src.bangladesh import get_glofas_data as ggd
 
+pd.options.mode.chained_assignment = None
+
 DATA_DIR = Path(os.environ["AA_DATA_DIR"])
 GLOFAS_DIR = DATA_DIR / "processed/bangladesh/GLOFAS_Data"
 STATION = "Bahadurabad_glofas"
 ffwc_dir = DATA_DIR / 'exploration/bangladesh/FFWC_Data'
 
+# Event definition
+EVENT_WATER_THRESH = 19.5 + 0.85
```

Comment:
Would it make sense for these kinds of parameters to go in a Bangladesh-specific ```config.yml```? Like in the case of mwi, for example?

Reply:
Yes definitely. When I change the GloFAS scripts to top-level, I will also make this change